tests/component-plugin/src/main/java/org/mule/functional/api/component/TestExceptionStrategy.java

```diff
@@ -1,49 +0,0 @@
-/*
- * Copyright (c) MuleSoft, Inc.  All rights reserved.  http://www.mulesoft.com
- * The software in this package is published under the terms of the CPAL v1.0
- * license, a copy of which has been included with this distribution in the
- * LICENSE.txt file.
- */
-
-package org.mule.functional.api.component;
-
-import static org.mule.runtime.core.api.util.ClassUtils.NO_ARGS;
-import static org.mule.runtime.core.api.util.ClassUtils.instantiateClass;
-import org.mule.runtime.core.api.Event;
-import org.mule.runtime.core.api.exception.MessagingException;
-import org.mule.runtime.core.api.exception.MessagingExceptionHandler;
-import org.mule.runtime.core.internal.exception.AbstractMessagingExceptionStrategy;
-
-/**
- * Provides a way to define custom exception strategies on test flows.
- *
- * @since 4.0
- */
-public class TestExceptionStrategy extends AbstractMessagingExceptionStrategy implements MessagingExceptionHandler {
```

Comment:
Extending from AbstractMessagingExceptionStrategy was the reason to make the test plugin a privileged one.
Remove it from the core's module privileged artifact list